src/emsdevice.cpp

```diff
@@ -831,29 +829,27 @@ void EMSdevice::generate_values_web_all(JsonArray & output) {
                     obj["v"]            = (divider > 0) ? time_value / divider : time_value * factor; // sometimes we need to divide by 60
                 }
             }
-        } else {
-            // must always have v for sorting to work in web
-            obj["v"] = "";
         }
 
-        // add name, prefixing the tag if it exists as the id (key for table sorting)
-        if (dv.full_name) {
-            if ((dv.tag == DeviceValueTAG::TAG_NONE) || tag_to_string(dv.tag).empty()) {
-                obj["id"] = dv.full_name;
-            } else {
-                char name[50];
-                snprintf(name, sizeof(name), "%s %s", tag_to_string(dv.tag).c_str(), read_flash_string(dv.full_name).c_str());
-                obj["id"] = name;
-            }
+        // id holds the shortname and must always have a value for the WebUI table to work
+        if (dv.tag >= DeviceValueTAG::TAG_HC1) {
+            obj["id"] = tag_to_string(dv.tag) + "/" + read_flash_string(dv.short_name);
         } else {
-            obj["id"] = "";
+            obj["id"] = read_flash_string(dv.short_name);
         }
 
-        // shortname
-        if (dv.tag >= DeviceValueTAG::TAG_HC1) {
-            obj["s"] = tag_to_string(dv.tag) + "/" + read_flash_string(dv.short_name);
-        } else {
-            obj["s"] = dv.short_name;
+        // n is the fullname, and can be optional
+        // don't add the fullname if its a command
+        if (dv.type != DeviceValueType::CMD) {
```

Comment:
Why do you skip the fullname for commands? I think show them as normal and delete line 842/843/853

Reply:
there is only one command, in the boiler called reset and it shows up in the Customizations as `reset (reset)` which is silly. The shortname is also not relevant as it's not part of the API and MQTT. If you think we still need to show the shortname then I would suggest changing the fullname to something more descriptive like 'reset command' ?

Reply:
But you skip the fullname for command and so it's handled as hidden, but it is not hidden, we want to have all options to show in web. Since you are using now the shortname as id, you can not skip the shortname.

Reply:
You can modify formatName() to 
```
    if (de.n === undefined || de.n === de.id) {
      return de.id;
```
to avoid "reset (reset)" and "pumpkick (pumpkick)", "activated (activated)", etc. This have no effect to the customization logic.

Reply:
Ok that's a much better option. Thanks

Reply:
I'm still not sending the entity fullname if its a command as this is the only way the Customizations web page knows whether it's a command or not.